### DISCUSSION TITLE
Error: 500: failed to create user account

### DISCUSSION THREAD

**Discussion by Kapsule-me** [AntlerVC/firetable#902] [Support / Q&A] [answered]
👉 **Please describe how to reproduce this bug here.**

I have no idea. I just get the following error message

![image](https://user-images.githubusercontent.com/104095124/198368185-7e4e7bbb-7dd8-4fbe-b085-bf9c955b8ac3.png)

**Comment by harinij** [comment]
@Kapsule-me Are you getting this error right away after logging into the rowy.app or creation of a workspace? It would be useful if you can share the steps from rowy.app that leads you to this error. Thanks

**Reply by Kapsule-me** [reply]
i had deleted a project and then tried to create a new project with the same firebase project and when it was created it sent me this error when i went back into my firestore and deleted the rowy collection the error went away.

appears to have been resolved.

**Comment by yoridewit** [comment, marked as answer]
Happens to me also. I already have a rowy account. This happens after succesful implementation of firebase security rules and general setup.
After autosetup completes and wants to take you to the rowy app, you get this message.

**Reply by shamsmosowi** [reply]
hi @yoridewit, did it resolve itself eventually, or are you still not able to access your project?

**Comment by yoridewit** [comment]
It apparently resolved itself eventually.

Op di 8 nov. 2022 om 22:02 schreef Shams ***@***.***>
